src/FastExpressionCompiler/ILReader.cs

```diff
@@ -613,21 +616,21 @@ public DynamicMethodILProvider(DynamicMethod method)
 
 public static class ILFormatter
 {
-    public static StringBuilder Int32ToHex(StringBuilder sb, int int32) => sb.Append(int32.ToString("X8"));
-    public static StringBuilder Int16ToHex(StringBuilder sb, int int16) => sb.Append(int16.ToString("X4"));
-    public static StringBuilder Int8ToHex(StringBuilder sb, int int8) => sb.Append(int8.ToString("X2"));
-    public static StringBuilder Argument(StringBuilder sb, int ordinal) => sb.Append($"V_{ordinal}");
-    public static StringBuilder Label(StringBuilder sb, int offset) => sb.Append($"IL_{offset:D4}");
+    public static StringBuilder Int32ToHex(StringBuilder sb, int int32) => sb.Append(int32.ToString("X8", CultureInfo.InvariantCulture));
+    public static StringBuilder Int16ToHex(StringBuilder sb, int int16) => sb.Append(int16.ToString("X4", CultureInfo.InvariantCulture));
+    public static StringBuilder Int8ToHex(StringBuilder sb, int int8) => sb.Append(int8.ToString("X2", CultureInfo.InvariantCulture));
+    public static StringBuilder Argument(StringBuilder sb, int ordinal) => sb.AppendFormat(CultureInfo.InvariantCulture, "V_{0}", ordinal);
+    public static StringBuilder Label(StringBuilder sb, int offset) => sb.AppendFormat(CultureInfo.InvariantCulture, "IL_{0:D4}", offset);
 
     public static StringBuilder MultipleLabels(StringBuilder sb, int[] offsets)
```

Comment:
Warnings could be ignored, but I think it's good to set the culture in case there is a side effect when rendered "technical" values like these.